lib/grizzly/zwave/commands/basic_get.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Grizzly.ZWave.Commands.BasicGet do
+  @moduledoc """
+  This module implements the BASIC_GET command form the COMMAND_CLASS_BASIC command class
+
+  Params: - none
+
+  """
+
+  @behaviour Grizzly.ZWave.Command
+
+  alias Grizzly.ZWave.Command
+  alias Grizzly.ZWave.CommandClasses.Basic
+
+  @impl true
+  def new(params) do
```

Comment:
can we do `new(params \\ [])` becuase otherwise the caller will always need to `new([])`?

Reply:
Forgot. Done.